iPSL/Electrical/Loads/Eurostag/package.mo

```diff
@@ -0,0 +1,18 @@
+within iPSL.Electrical.Loads;
+package Eurostag "Load models from Eurostag components"
+annotation (Documentation(info="<html>
+<p><br><span style=\"font-family: MS Shell Dlg 2;\">&LT;iPSL: iTesla Power System Library&GT;</span></p>
```

Comment:
Unfortunately Dymola 2016 adds this span "spam". It's the WYSIWYG editor that does this. This can be easily cleaned up by marking the whole text in the WYSIWYG editor and set the font back to `(default)`. This is something that should probably also be done in the rest of the library but best to catch as early as possible.